tests/Queries.elm

```diff
@@ -206,21 +207,61 @@ testFirst output =
 testIndex : Single msg -> Test
 testIndex output =
     describe "Query.index"
-        [ describe "Query.index 0" <|
-            [ test "sees it's a <div class='container'>" <|
-                \() ->
+        [ describe "only 1 element"
+            [ fuzz (Fuzz.intRange -1 0) "matches" <|
+                \index ->
                     output
                         |> Query.findAll []
-                        |> Query.index 0
+                        |> Query.index index
                         |> Query.has [ tag "div", class "container" ]
-            ]
-        , describe "Query.index -1" <|
-            [ test "sees it's a <div class='container'>" <|
-                \() ->
+            , fuzz (Fuzz.intRange -3 -2) "too low" <|
+                \index ->
                     output
                         |> Query.findAll []
-                        |> Query.index -1
-                        |> Query.has [ tag "div", class "container" ]
+                        |> Query.index index
+                        |> Query.hasNot [ tag "div" ]
+            , fuzz (Fuzz.intRange 1 2) "too high" <|
+                \index ->
+                    output
+                        |> Query.findAll []
+                        |> Query.index index
+                        |> Query.hasNot [ tag "div" ]
+            ]
+        , describe "only 3 element"
+            [ fuzz (Fuzz.intRange -3 2) "matches" <|
```

Comment:
@rtfeldman is there a way to write a test for a list of values? e.g. there is no need for fuzzing if you know all the values you wanna test, right?

```elm
testFor [-3, -2, -1, 0, 1, 2] "matches" <|
  \index ->
-- or
testFor (List.range -3 2) "matches" <|
  \index ->
```